packages/icons/README.md

```diff
@@ -0,0 +1,26 @@
+# @trezor/icons
+
+This package contains fully-fledged icon components using the Skia engine.
+
+Icon components are ready to use only with React Native app now, but it is planned to be used by the web in the future as well.
+
+Components and their definitions are split out into two separate files:
+
+`Component.tsx` is picked up by Metro bundler for both Android and iOS platforms.
+
+`Component.web.tsx` is picked up by Webpack bundler for web.
+
+## How to add or update icon
+
+1. Export icon as SVG from Figma (no other types than SVG are allowed).
+2. Rename icon to follow camel case convention (Warning Circle.svg => warningCircle.svg).
```

Comment:
I think we should add step where we describe you should put icon somewhere to assets folder